app/javascript/styles/_task-list.scss

```diff
@@ -13,7 +13,6 @@
 
 .app-task-list__section {
   display: table;
-  @include govuk-font($size:24, $weight: bold);
```

Comment:
I think we can leave this, at least for now, as the task list section heading sizes shouldn’t need to change.

Reply:
I think that the size gets set by adding `govuk-heading-m` to the element, so this include is redundant. Did you mean that we should bring it back or is it ok as is?

Reply:
Yup, I would revert this change